Examples/UIExplorer/UIExplorerApp.js

```diff
@@ -29,9 +30,30 @@ var UIExplorerApp = React.createClass({
   getInitialState: function() {
     return {
       openExternalExample: (null: ?React.Component),
+      tintColor: '#008888',
+      barTintColor: null,
+      titleTextColor: null,
     };
   },
 
+  _customNavColors: function() {
+    this.setState({
+      tintColor: '#FFFFFF',
+      barTintColor: '#4F6BA5',
+      titleTextColor: '#FFFFFF',
+    });
+    StatusBarIOS.setStyle(StatusBarIOS.Style['lightContent']);
```

Comment:
Lets remove the StatusBarIOS code from here. There is already an example for that. If we keep every example as simple as possible, people will be able to learn faster.

Also, with our style conventions, this would look like this: `StatusBarIOS.Style.lightContent`